pkg/collector/helm_collector.go

```diff
@@ -0,0 +1,86 @@
+package collector
+
+import (
+	"github.com/Azure/aks-periscope/pkg/interfaces"
+	"github.com/Azure/aks-periscope/pkg/utils"
+	"path/filepath"
+)
+
+// HelmCollector defines a Helm Collector struct
+type HelmCollector struct {
+	BaseCollector
+}
+
+var _ interfaces.Collector = &HelmCollector{}
+
+// NewHelmCollector is a constructor
+func NewHelmCollector(exporter interfaces.Exporter) *HelmCollector {
+	return &HelmCollector{
+		BaseCollector: BaseCollector{
+			collectorType: Helm,
+			exporter:      exporter,
+		},
+	}
+}
+
+// Collect implements the interface method
+func (collector *HelmCollector) Collect() error {
+	rootPath, err := utils.CreateCollectorDir(collector.GetName())
+	if err != nil {
+		return err
+	}
+
+	_, err = utils.RunCommandOnContainer("apk", "add", "curl", "openssl", "bash", "--no-cache")
+	if err != nil {
+		return err
+	}
+
+	_, err = utils.RunCommandOnContainer("curl", "-fsSl", "-o", "get_helm.sh", "https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3")
+	if err != nil {
+		return err
+	}
+
+	_, err = utils.RunCommandOnContainer("chmod", "+x", "get_helm.sh")
+	if err != nil {
+		return err
+	}
+
+	_, err = utils.RunCommandOnContainer("./get_helm.sh")
+	if err != nil {
+		return err
+	}
+
+	helmListFile := filepath.Join(rootPath, "helm_list")
+	output, err := utils.RunCommandOnContainer("helm", "list", "--all-namespaces")
```

Comment:
💡 🐛 There are few things not going to work with this `helm` command, and I will discuss in detail with some suggestions, since its a daemon set, we could achieve similar idea with `kubectl` I will discuss this with right folks before moving forward.

In the mean time, can you test this in any `ARC-Enabled` cluster as I believe logs will be empty every time. There is a`kubectl` command which can get right data for it.